api/src/main/java/org/pmiops/workbench/api/OfflineRdrExportController.java

```diff
@@ -63,14 +68,15 @@
       try {
         groupIdsAndPushTask(rdrExportService.findAllUserIdsToExport(), EXPORT_RESEARCHER_PATH);
       } catch (Exception ex) {
-        log.severe(
-            String.format("Error while exporting researcher data to RDR: %s", ex.getMessage()));
+        throw new ServerErrorException(
```

Comment:
In a controller you might want to throw a more specific error here and let the handler translate it into a 500, but I'm not sure if that happens with crons. This is fine.